about.html

```diff
@@ -0,0 +1,104 @@
+                <li><a href="" class="menu-links">Program</a></li>
+                <li><a href="" class="menu-links">Join</a></li>
+                <li><a href="" class="menu-links">Sponsors</a></li>
+                <li><a href="index.html" class="menu-links">Project X</a></li>
+            </ul>
+            <nav class="navbar-about">
+                <img src="sources/mobile-version/images/tiny-logo.png" alt="" class="logo">
+                <img src="sources/mobile-version/images/hamburger-button.png" alt="" class="hamburger">
+            </nav>
+            <h2 class="header-h2-title">"Hello! Project X"</h2>
+            <h1 class="header-h1-title">PROJECT X. A NEW EXPERIENCE IN 2023</h1>
+        </div>
+        <div class="container-p">
+            <p class="header-p">We are a little startup with the buisness objective of get to know in the world as a provider of event as service. In the field of the music and the celebrities</p>
+        </div>
+        <p class="email">
+            Please contact us per Email for any further questions about Project X! <br>
+            <a href="">projectxstaff@hotmail.com</a>
+        </p>
+    </header>
+    <main class="main">
+        <h2>Project X Logo</h2>
+        <div class="orange-line"></div>
+        <h3>The logo was made in a house party in the day of the fraternities</h3>
+        <div class="logo-container">
+            <img src="sources/mobile-version/images/logo.png" alt="">
+        </div>
+        <hr>
+        <h2>See a little bit of our story until now</h2>
+        <div class="orange-line"></div>
+        <h3>Take a lok at the least 2 partys we organized</h3>
+        <div class="desktop-background-image-container">
+            <div class="background-image">
+                <div class="overlay"></div>
+                <h2 class="header-h2">"Hello! Project X"</h2>
+                <h1 class="header-h1">PROJECT X. A NEW EXPERIENCE IN 2020</h1>
+            </div>
+            <div class="background-image">
+                <div class="overlay-2"></div>
+                <h2 class="header-h2">"Hello! Project X"</h2>
+                <h1 class="header-h1">PROJECT X. A NEW EXPERIENCE IN 2019</h1>
+            </div>
+        </div>
+        <div class="partners-container">
+            <h2>Partner</h2>
+            <div class="orange-line"></div>
+            <div class="partners2">
+                <img src="sources/images/obey-logo.png" alt="" class="partner">
+                <img src="sources/images/death-row-records-logo.png" alt="" class="partner">
+                <img src="sources/images/shenheiser-logo.png" alt="" class="partner">
+                <img src="sources/images/tomorrowland-logo.png" alt="" class="partner">
+            </div>       
+        </div>
+        <footer>
+            <img src="sources/mobile-version/images/logo.png" alt="">
+            <div>
+                <h3>2023 Project X enterprises, UK.</h3>
```

Comment:
- [ ] like the previous reviewer noted, kindly centralize your content of the footer in the about page. The logo to the left and text to the right. 
- [optional] kindly add the height of the footer section so it can match the original design.
![Screenshot (1174)_LI](https://github.com/SaveryIV/First-Capstone-Project/assets/65298048/1475e947-cddb-49ee-a534-b26d7511af71)

Reply:
The first reviewer says me the position i put it is right